app/src/app/components/BrushControls.tsx

```diff
@@ -5,16 +5,19 @@ import PaintByCounty from './Toolbar/PaintByCounty';
 import {ZonePicker} from './Toolbar/ZonePicker';
 export const BrushControls = () => {
   const activeTool = useMapStore(state => state.activeTool);
+  const mapType = useMapStore(state => state.mapDocument?.map_type);
```

Comment:
I: By using the `mapType` value inside these components, we may create a ton of complexity / conditional logic. What do you think about taking the following approach:
- New store for feature flags
- Manage all feature flags as subscriptions on the map type
- Only consume descriptive boolean feature flags in components, e.g. `{ paintByCounty ?? <PaintByCounty /> }` rather than `{ mapType !== 'local' ?? <PaintByCounty /> }`

```suggestion
  const paintByCounty = useFeatureFlagStore(state => state.paintByCounty);
```

This design would have the added advantage as also working for user-set feature flags / preferences. This could be useful in cases where we want to test out a new feature but not turn it on for any map types by default.

Reply:
+1 

For updating state we can subscribe to `mapStore.mapDocument` and update `featureFlagStore` on change, separating the concerns of any feature logic there

Reply:
ok this was my first time adding in a store / subscription but I think this is working